api.py

```diff
@@ -249,3 +253,96 @@ def upload_task_status_update(req: UpdateTaskStatusRequest) -> None:
     tdjson = res.json()
     if "errors" in tdjson:
         raise Exception(json.dumps(tdjson.get("errors")))
+
+
+def validate_json(json_data, json_schema):
+    try:
+        jsonschema.validate(instance=json_data, schema=json_schema)
+    except jsonschema.exceptions.ValidationError as err:
+        log.error(msg=err)
```

Comment:
is this the correct way to log errors? ive been doing this `log.error(err)`

Reply:
affectively the same thing